client/logger

```diff
@@ -0,0 +1 @@
+../cloud/logger
```

Comment:
Angular repo track client packages only. Of we want this we will need to extend that to cloud which has some problems.